installer/sharch_body.sh

```diff
@@ -25,13 +25,20 @@ fi
 
 echo " OK."
 
+image_size=$(du "$0" | awk '{print $1}')
 # Untar and launch install script in a tmpfs
 cur_wd=$(pwd)
 export cur_wd
 archive_path=$(realpath "$0")
 tmp_dir=$(mktemp -d)
 if [ "$(id -u)" = "0" ] ; then
     mount -t tmpfs tmpfs-installer $tmp_dir || exit 1
+    mount_size=$(df $tmp_dir | tail -1 | tr -s ' ' | cut -d' ' -f4)
+    if [ "$mount_size" -lt "$((image_size*3))" ]; then
+        #align mount_size to next power of two
+        mount_size=$(echo $((image_size*3)) | awk '{ print 2^int(((log($1)/log(2))+ 1)) }')
```

Comment:
Dont complicate with math logic.
Use dictionry/array, to select numbers 8,16,32,64,128

Reply:
Can u change logic to 
image_size=`du  target/sonic-marvell-armhf.bin | awk '{print $1}'`
mount_size=$((((image_size*3)/1000/1000)+1))
mount -o remount,size="${mount_size}G" -t tmpfs tmpfs-installer $tmp_dir || exit 1

Reply:
okay